src/theory/sets/inference_manager.h

```diff
@@ -53,19 +53,26 @@ class InferenceManager : public InferenceManagerBuffered
    *
    * The argument c is the name of the inference, which is used for debugging.
    */
-  void assertInference(Node fact, Node exp, const char* c, int inferType = 0);
+  void assertInference(Node fact,
```

Comment:
We can probably drop `c` from this method now, as it is subsumed by the id.  Similar to all of the below methods.